lib/module.js

```diff
@@ -253,9 +254,22 @@ if (process.platform === 'win32') {
       // Use colon as an extra condition since we can get node_modules
       // path for dirver root like 'C:\node_modules' and don't need to
       // parse driver name.
-      if (code === 92/*\*/ || code === 47/*/*/ || code === 58/*:*/) {
-        if (p !== nmLen)
-          paths.push(from.slice(0, last) + '\\node_modules');
+      if (code === 92/*\*/ || code === 47/*/*/ || code === 58/*:*/
+        || (adjacentNodeModules && code === 46/*.*/)) {
```

Comment:
Operators go at the end of the previous line. Also, the next line should match the indentation of the start of the first argument on the previous lines.